cpp/test/neighbors/ann_ivf_pq.cuh

```diff
@@ -21,8 +21,7 @@
 #include "naive_knn.cuh"
 #include <cuvs/neighbors/ivf_pq.hpp>
 
-#include <raft/neighbors/ivf_pq-inl.cuh>
-#include <raft/neighbors/sample_filter.cuh>
+#include <cuvs/neighbors/sample_filter.hpp>
```

Comment:
IVF flat tests need to be moved over and they should only be invoking the public APIs exposed through `include/`.